src/components/Menu/menuItemGroup.tsx

```diff
@@ -0,0 +1,12 @@
+import { styled } from '@mui/material';
+
+export const EnteMenuItemGroup = styled('div')`
+    & > :not(:last-of-type) {
```

Comment:
> & > :not(:last-of-type) {
    
can update it to 
```& > .MuiMenuItem-root:not(:last-of-type) {```

so that later on, if we change something, we could be sure that `MuiMenuItem-root` radius is affected